Stylesheets/pastelSolarized.nb

```diff
@@ -0,0 +1,287 @@
+(* Content-type: application/vnd.wolfram.mathematica *)
```

Comment:
While we are at it, should we clear the notebooks from the Cache, Window position, etc.. entries?